packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -31,10 +31,10 @@ type WorkflowAndFolderUnion = {
 	updatedAt: Date;
 };
 
-export type WorkflowAndFolderUnionFull = (
+export type WorkflowAFolderUnionFull = (
```

Comment:
Should this be either `WorkflowFolderUnionFull` or `WorkflowAndFolderUnionFull`?